scoring/score.py

```diff
@@ -1,20 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import itertools
 import collections
 
 
 class InvalidScoresheetException(Exception):
     pass
 
 
+POINTS_HELD = 1
+POINTS_GROUND = 2
+POINTS_PLATFORM = 5
+
+NUM_TOKENS = 16
```

Comment:
Really obnoxious point, but perhaps "num tokens in arena"? That's mostly obvious from context because it's a constant but there are multiple counts of tokens at play at various points in this code.

Reply:
4031047